src/tests/contract_test/wallet_test.cljs

```diff
@@ -65,3 +60,18 @@
     (fn []
       (p/let [response (contract-utils/call-rpc "wallet_getEthereumChains")]
         (assert-ethereum-chains response)))))
+
+(defn assert-wallet-tokens
+  [tokens]
+  (let [flattened-tokens (mapcat val tokens)]
+    (is (some :symbol flattened-tokens))))
```

Comment:
I really think we should be testing more than this here. 
Can you take a look through the code and see what properties of the tokens are being used and then ensure these are captured??
cc @ulisesmac, @smohamedjavid perhaps you could point to some uses of the data we are dependent on that we get from this rpc endpoint ("wallet_getWalletToken")

Reply:
Ideally, we should always receive the `:symbol` not being an empty string, but I remember we received some tokens named `""` :disappointed: 

other important stuff is the Wei amount and `:decimals`, since we derive info from it.

Reply:
The whole token data depends on it 😅. 
Keys of a token that we use from `wallet_getWalletToken` RPC.
```clojure
{:symbol "ETH"
 :name "Ether"
 :decimals 18
 :balances-per-chain {1 {:raw-balance "193823235482934892"}
                      10 {:raw-balance "000004934800028760"}
                      ...}
 :market-values-per-currency {:usd {:price 3305.35 :change-pct-24hour -6.053133328320746}}}
                                            
;; NOTE: 1 and 10 are the chain ids of Ethereum mainnet and Optimism respectively. This would change if the user has enabled testnet.
```

Reply:
Im pushing a new version. Do we need the ` :market-values-per-currency`? Its the only one Im not testing

Reply:
@FFFra  Yeah, we need it. we use it to calculate  the fiat balance.

Reply:
Yes, we need to test it, as it is one of the essential keys used to calculate and display the fiat balance.
Just a check whether the keys `market-values-per-currency` and `:usd` (default currency) are present should suffice. (Because for some tokens (e.g. a bridged token), we receive 0 as value. Not needed to go deep)